METdbLoad/ush/read_data_files.py

```diff
@@ -1141,7 +1141,8 @@ def get_lookup(filename):
             lu_type = CN.STAT
         elif lc_filename.endswith(".vsdb"):
             lu_type = CN.VSDB_POINT_STAT
-        elif lc_filename.endswith("cts.txt"):
+        elif (lc_filename.endswith("cts.txt") and 
+              "/mode" in lc_filename):
```

Comment:
Thanks @TatianaBurek for testing that this changes works. I do wonder if there's a way to make this check even more explicit? What if the string "/mode" exists somewhere further up in the path of a file named "grid_stat*cts.txt"? That'd break this logic.

Is it possible to do something like:
```
        elif (lc_filename.endswith("cts.txt") and 
              lc_filename.basename().beginswith("mode"):
```
But I don't know the correct syntax for "basename" and "beginswith".

Reply:
And I actually just thought of an example of this. In the context of METplus, some use cases are organized into a "model_applications" sub-directory. I don't have any data on hand to test and demonstrate but it does seem that a "model_applications" sub-directory would likely match this!
```
"/mode" in lc_filename
```

Reply:
I'll use os.path.basename(lc_filename) with beginswith() so we can avoid that case.